README.rst

```diff
@@ -145,6 +145,11 @@ the default cwl-runner use::
 
   cwltool [tool-or-workflow-description] [input-job-settings]
 
+You can set cwltool options in the environment with CWLTOOL_OPTIONS,
+these will be inserted at the beginning of the command line::
+
+  export CWLTOOL_OPTIONS="--debug"
+
```

Comment:
Can you pull this out to a separate PR?

Reply:
Sure

Reply:
https://github.com/common-workflow-language/cwltool/pull/1205